spyder/plugins/editor/lsp/manager.py

```diff
@@ -300,6 +300,12 @@ def generate_python_config(self):
             'enabled': self.get_option('jedi_definition'),
             'follow_imports': self.get_option('jedi_definition/follow_imports')
         }
+        jedi_hover = {
+            # This option needs to be True always so that we can request
+            # information for the Object Inspection Help Pane
+            # TODO: Something similar might be needed for other languages 
+            'enabled': True,
+        }
```

Comment:
There's no need to have this here because this value is already `True` in `PYTHON_CONFIG`. But you can move the comment above there.